### PR TITLE
Initial description of TSC

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

**Comment by karianna** [comment]
LGTM as a starting point but I expect a lot of people will want to discuss even this opening PR in detail.  Let's throw it to the #getopenjdk Slack channel and the adoption-discuss mailing list.

**Review by a-roberts — COMMENTED** [review]
I propose

1) mandatory reelection process annually - to reduce complacency and encourage folks to want to be on the TSC
2) candidates for the committee should mention their ideas and intentions for the project

Draft wording would be 

The appointing of TSC members occurs on an annual basis and is based on a meritocracy. Candidates should be forthcoming with ideas for the project and welcoming to contributors (particularly new ones). Candidates with the intention of becoming on the TSC should briefly outline where they'd like to see the project going - all in a transparent manner that is available to the public.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,50 @@
 # TSC
-The AdoptOpenJDK Technical Steering Comittee 
+The AdoptOpenJDK Technical Steering Comittee is the technical governing body of the AdoptOpenJDK organisation. It admits and oversees all top-level Projects in the AdoptOpenJDK organisation.
```
Comment (tellison) [inline README.md]: Based on the following list, are you equating "top level projects" with repositories in the org?
I guess we have to be aware of the difference / overlap / relationship between the Github org and the OpenJDK adoptopenjdk group.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,50 @@
...
+* https://github.com/AdoptOpenJDK/openjdk-api
+* https://github.com/AdoptOpenJDK/openjdk-nightly
+* https://github.com/AdoptOpenJDK/openjdk-releases
+* https://github.com/AdoptOpenJDK/openjdk-jdk8u
+* https://github.com/AdoptOpenJDK/openjdk-jdk9
+* https://github.com/AdoptOpenJDK/openjdk-infrastructure
+* https://github.com/AdoptOpenJDK/openjdk-website-backend
+* https://github.com/AdoptOpenJDK/secrets _private_
+
+### List of TSC Responsibilities
```
Comment (tellison) [inline README.md]: While these responsibilities are good to see listed, I would hope that we are open to anyone coming to the project and contributing in these areas.  The goal should be to work in the open, and achieve consensus with all participants based on technical merit wherever possible.  I would not expect anyone to "flex their TSC credentials".

File: README.md [hunk 3]
```diff
@@ -1,2 +1,50 @@
...
+* Setting and maintaining standards covering contributions of code, documentation and other materials
+* Managing code and binary releases: types, schedules, frequency, delivery mechanisms
+* Creating new repositories and projects under the _AdoptOpenJDK_ GitHub organisation as required
+* Setting overall technical direction for the AdoptOpenJDK organisation, including high-level goals and low-level specifics regarding features and functionality
+* Setting and maintaining appropriate standards for community discourse via the various mediums under TSC control
+
+## TSC Members
+
+TSC members can nominate new members at any time. Candidates for membership tend to be people
+who have a competancy for community management and a high tolerance and patience for process
```
Comment (tellison) [inline README.md:38]: Can you explain "high tolerance and patience for process minutiae"?  I was expecting the TSC to be a *technical* merit body to guide the best interests of the org/projects.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,50 @@
...
+Every Dependant Project not currently incubating can appoint someone to the TSC who they elect
+at their own discretion.
+
+### Current Members
+| Avatar | Information |
+|---|:---|
+|   |    |
+
+## Rules for TSC Membership
+* No single company can occupy more than 1/3 seats on the TSC
```
Comment (tellison) [inline README.md]: Is this necessary?  People should be working in the interests of the project, not their company.  If we see people pushing selfish corporate goals we'd need to address that promptly anyway.

What happens if the TSC becomes imbalanced, do we draw lots to kick people off?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,50 @@
...
+* The appointing of TSC members occurs on an annual basis and is based on a meritocracy.
```
Comment (tellison) [inline README.md]: Is the full card appointed each year, or people serve a one-year term?  Would be good to have some continuity of experience.

**Review by tellison — COMMENTED** [review]
Thanks for starting this description, it raises a number of good questions.

I've added a few thoughts as separate comments to the doc.

File: README.md (hunk 4 above)
Comment (gdams) [reply README.md]: well when one member leaves we would simply re-elect but consider the balance

**Review by karianna — APPROVED** [review]
Good for a starting point to discuss - we also want to build out the knowledge base from here, hence merging.

**Pull request merged**